docs/user/customization.md

```diff
@@ -64,4 +64,167 @@ An example `worker.ign` is shown below. It has been modified to increase the HTT
+    -rw-r--r--. 1 xxxxx xxxxx 2655 Feb 28 10:54 99_openshift-cluster-api_worker-user-data-secret.yaml
+    ```
+
+2. Create a `MachineConfig` that includes `kubelet.service` that has no taints.
+
+    ```sh
+    cat > no-taint-cluster/openshift/99-master-kubelet-no-taint.yaml <<EOF
+    apiVersion: machineconfiguration.openshift.io/v1
+    kind: MachineConfig
+    metadata:
+      labels:
+        machineconfiguration.openshift.io/role: master
+      name: 02-master-kubelet
+    spec:
+      config:
+        ignition:
+          version: 2.2.0
+        systemd:
+          units:
+          - contents: |
+              [Unit]
+              Description=Kubernetes Kubelet
+              Wants=rpc-statd.service
+
+              [Service]
+              Type=notify
+              ExecStartPre=/bin/mkdir --parents /etc/kubernetes/manifests
+              ExecStartPre=/bin/rm -f /var/lib/kubelet/cpu_manager_state
+              EnvironmentFile=-/etc/kubernetes/kubelet-workaround
+              EnvironmentFile=-/etc/kubernetes/kubelet-env
+
+              ExecStart=/usr/bin/hyperkube \
+                  kubelet \
+                    --config=/etc/kubernetes/kubelet.conf \
+                    --bootstrap-kubeconfig=/etc/kubernetes/kubeconfig \
+                    --rotate-certificates \
+                    --kubeconfig=/var/lib/kubelet/kubeconfig \
+                    --container-runtime=remote \
+                    --container-runtime-endpoint=/var/run/crio/crio.sock \
+                    --allow-privileged \
+                    --node-labels=node-role.kubernetes.io/master \
+                    --minimum-container-ttl-duration=6m0s \
+                    --client-ca-file=/etc/kubernetes/ca.crt \
+                    --volume-plugin-dir=/etc/kubernetes/kubelet-plugins/volume/exec \
+                    \
+                    --anonymous-auth=false \
+
+              Restart=always
+              RestartSec=10
+
+              [Install]
+              WantedBy=multi-user.target
+            enabled: true
+            name: kubelet.service
+    EOF
+    ```
+
+    `machineconfiguration.openshift.io/role: master` label attaches this `MachineConfig` to the [master][master-machine-config-pool] `MachineConfigPool`. The [default][default-kubelet-service] configuration for the `kubelet.service` on libvirt includes the taint.
+
+3. Run `cluster` target to create the cluster using the custom manifests.
+
+    ```console
+    $ openshift-install --dir no-taint-cluster create cluster
+    INFO Consuming "Openshift Manifests" from target directory
+    INFO Consuming "Master Machines" from target directory
+    INFO Consuming "Common Manifests" from target directory
+    INFO Creating cluster...
+    INFO Waiting up to 30m0s for the Kubernetes API...
+    ...
+    ```
+
+    ```console
+    $ oc --config no-taint-cluster/auth/kubeconfig get nodes -ojson | jq '.items[] | select(.metadata.labels."node-role.kubernetes.io/master" == "") | .spec.taints'
```

Comment:
Add a line about what this is doing (esp bc this is half hidden in my browser so the full command isn't visible without lots of scrolling).

Reply:
done

Reply:
Perfecto!!